### PR TITLE
Added burrito command, made food commands use JSON

### LINKED ISSUE #249: No burrito command
This is a very very very minor bug, but I think that there should be a command for burrito

### PR DESCRIPTION
**Pull request by dignissimus**
Added the burrito command (Resolves #249) and made the majority of food related commands use json files instead of formatting strings.

### CODE REVIEW

**Comment by Proximyst** [comment]
:pray:

File: data/cake.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            "makes",
+            "gives",
+            "gets",
+            "buys"
+        ],
+        "side": [
+            "glass of chocolate milk",
+            "bowl pf ice cream",
+            "jar of cookies",
+            "some chocolate sauce"
```
Comment (perryprog) [inline data/cake.json]: "and serves it with a small some chocolate sauce" wouldn't make much sense. 🙁

File: data/cake.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
+    "templates": [
+        "{method} {user} a {flavor} {size} {type} cake and serves it with a small {side}!"
```
Comment (daboross) [inline data/cake.json]: This file is missing `{side}`.

File: data/cake.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        "method": [
+            "bakes",
+            "makes",
+            "gives",
+            "gets",
+            "buys"
+        ],
+        "side": [
+            "glass of chocolate milk",
+            "bowl pf ice cream",
```
Comment (daboross) [inline data/cake.json]: `pf` -> `of`

File: data/cookie.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            "tasty",
+            "delectabel",
+            "delicious",
+            "yummy",
+            "toothsome",
+            "scrumptious",
+            "luscious"
+        ]
+    },
+    "method": [
```
Comment (daboross) [inline data/cookie.json]: `method` and `side` are outside `parts` incorrectly.

File: data/cookie.json [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        "size": [
+            "small",
+            "little",
+            "medium-sized",
+            "large",
+            "gigantic"
+        ],
+        "flavor": [
+            "tasty",
+            "delectabel",
```
Comment (daboross) [inline data/cookie.json]: `delectabel` -> `delectable`

**Pull request merged**